lib/checkMergeConflicts.js

```diff
@@ -68,18 +73,21 @@ module.exports.checkMergeConflictsInPullRequest = async function(context, pullRe
   }
 };
 
+/**
+ * @param {import('probot').Context} context
+ */
 module.exports.checkMergeConflictsInAllPullRequests = async function(context) {
-  // pullRequests.getAll(..) fetches atmost 30 PRs in a batch, by default.
+  // pullRequests.list(..) fetches atmost 30 PRs in a batch, by default.
```

Comment:
This should be `pulls.list` (OR `context.github.pulls.list`)

Reply:
Ahh
I missed it.
Thanks